app/src/main/java/chat/rocket/android/util/TokenSerialisableModel.kt

```diff
@@ -0,0 +1,36 @@
+package chat.rocket.android.util
+
+import java.io.Serializable
+
+class TokenSerialisableModel(
+    private val firstParam: String,
+    private val secondParam: String,
+    private val thirdParam: String = ""
+) :
+    Serializable {
+    companion object {
```

Comment:
`companion object` should stay at the bottom of the class - by the code style used.